content/guides/guides/module-api.md

```diff
@@ -45,29 +45,29 @@ node e2e-run-tests.js
+| `headless`          | _boolean_ | Hide the browser instead of running headed (default during `cypress run`)                                                                                          |
+| `key`               | _string_  | Specify your secret record key                                                                                                                                     |
+| `exit`              | _boolean_ | Whether to close Cypress after all tests run                                                                                                                       |
+| `parallel`          | _boolean_ | Run recorded specs in [parallel](/guides/guides/parallelization) across multiple machines                                                                          |
+| `port`              | _number_  | Override default port                                                                                                                                              |
+| `project`           | _string_  | Path to a specific project                                                                                                                                         |
+| `quiet`             | _boolean_ | If passed, Cypress output will not be printed to `stdout`. Only output from the configured [Mocha reporter](/guides/tooling/reporters) will print.                 |
+| `record`            | _boolean_ | Whether to record the test run                                                                                                                                     |
+| `reporter`          | _string_  | Specify a [Mocha reporter](/guides/tooling/reporters)                                                                                                              |
+| `reporterOptions`   | _object_  | Specify [Mocha reporter](/guides/tooling/reporters) options                                                                                                        |
+| `slowTestThreshold` | _number_  | Time, in milliseconds, to consider a test "slow" during `cypress run`. A slow test will display in orange text in the default reporter.                            |
+| `spec`              | _string_  | Specify the specs to run, see examples below                                                                                                                       |
+| `tag`               | _string_  | Identify a run with a tag or tags                                                                                                                                  |
+| `testingType`       | _string_  | Specify the type of tests to execute; either `e2e` or `component`. Defaults to `e2e`                                                                               |
 
 ### Examples
```

Comment:
at the bottom of this page, Do we need to add a history entry about this value changing to be testing-type spcific in 10.x?

Reply:
I wasn't sure, because nothing else about 10.0 had been added. The fact that no one else has before isn't a good reason not to, though, so added it in.

It seems possible that we'll end up supersceding the list of specific changes with a "Reworked everything; See 10.x changelog for details" link by the time we're through.

Reply:
I created a story for DX to add the other 10 config changes to the history of these files